lib/core/components/MantraForm.js

```diff
@@ -56,4 +71,21 @@ export default class MantraForm extends MantraComponent {
         const filteredInfo = pathInfo.filter((str) => str);
         return filteredInfo.join('.');
     };
+
+    /**
+     * Verify the validity of the MantraForm constructor params
+     * @param {Object} value - The constructor params
+     */
+    _isConstructorValid({ model }) {
```

Comment:
I think what would be grammatically correct is: 
```suggestion
    _isValidConstructor({ model }) {
```